server/routes/pokeDex.js

```diff
@@ -0,0 +1,45 @@
+const { response } = require("express");
+const express = require("express");
+const router = express.Router();
+
+const axios = require("axios");
+
+router.get("/", async (req, res) => {
+   try {
+      const getPokeDexOffset = async () => {
+         try {
+            return await axios.get(
```

Comment:
Where is this package coming from? I don't see it in your `package.json` 🤔

Reply:
Ah, I might have downloaded Axios globally from past projects, and therefore hasn't given me an error. I will have this package to avoid this issue in my next commit.